pkg/kubelet/rkt/fake_rkt_interface.go

```diff
@@ -55,11 +56,40 @@ func (f *fakeRktInterface) GetInfo(ctx context.Context, in *rktapi.GetInfoReques
 }
 
 func (f *fakeRktInterface) ListPods(ctx context.Context, in *rktapi.ListPodsRequest, opts ...grpc.CallOption) (*rktapi.ListPodsResponse, error) {
-	return nil, fmt.Errorf("Not implemented")
+	f.Lock()
+	defer f.Unlock()
+
+	f.called = append(f.called, "ListPods")
+
+	var result []*rktapi.Pod
+	for _, p := range f.pods {
```

Comment:
maybe we can just record the filters, so for GetPods(false) and GetPods(true) we only need to check that the filters are correct?